cfgov/paying_for_college/tests/test_documents.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+
+from paying_for_college.documents import SchoolDocument
+from paying_for_college.models import School
+
+
+class SchoolDocumentTest(TestCase):
+
+    def test_prepare(self):
+        school = School(school_id="999999",
+                        settlement_school="Example University",
+                        city="Example City", state="VA")
+        doc = SchoolDocument()
+
+        prepared_data = doc.prepare(school)
+        self.assertEqual(prepared_data, {
+            'autocomplete': doc.prepare_autocomplete(school),
```

Comment:
I don't think we should be calling `doc.prepare_*` in our test assertions for the expected results, we can hard code them, or reference them directly from the school object. This test would be more testing that the `prepare` function calls each defined fields `prepare` function, rather than what we want to test which is that the prepare function properly maps all fields from our django model to the document class.

Reply:
Removed doc.prepare_city and doc.prepare_state from test assertion.
The city and state are now referenced from the school object.